workshop_app/urls.py

```diff
@@ -29,9 +29,12 @@
     url(r'^logout/$', views.user_logout),
     url(r'^view_profile/$', views.view_profile),
     url(r'^edit_profile/$', views.edit_profile),
-    url(r'^my_workshops/$', views.my_workshops),
+    url(r'^my_workshops/$', views.my_workshops, name='my_workshops'),  # Coordinator
```

Comment:
Please refactor these patterns to reflect the new Django URL patterns - see https://docs.djangoproject.com/en/3.0/topics/http/urls/#example

Reply:
The URL name field should be clear and descriptive enough such that you do not need the comments to tell you what the URL does!